ghost/core/core/server/models/member.js

```diff
@@ -46,6 +46,13 @@ const Member = ghostBookshelf.Model.extend({
         }, {
             key: 'conversion',
             replacement: 'conversions.attribution_id'
+        }, {
+            key: 'clicked_links',
+            replacement: 'clicked_links'
```

Comment:
Why are we replacing like for like here?

Reply:
Hmmm... Just tested it again without this rule and it works. I added this rule (which indeed is a bit weird) because it wasn't transforming the `clicked_links` to the proper name in the query (and got errors from that). Maybe a temporary issue with code that wasn't reloaded properly. Removed it now ✅